include/nbl/video/SPhysicalDeviceFeatures.h

```diff
@@ -0,0 +1,508 @@
+    //VkBool32    sampleRateShading;
+    //VkBool32    dualSrcBlend;
+    bool logicOp = false;
+    bool multiDrawIndirect = false;
+    bool drawIndirectFirstInstance = false;
+    bool depthClamp = false;
+    bool depthBiasClamp = false;
+    bool fillModeNonSolid = false;
+    bool depthBounds = false;
+    bool wideLines = false;
+    bool largePoints = false;
+    bool alphaToOne = false;
+    bool multiViewport = false;
+    bool samplerAnisotropy = false;
+    //VkBool32    textureCompressionETC2;
+    //VkBool32    textureCompressionASTC_LDR;
+    //VkBool32    textureCompressionBC;
+    //VkBool32    occlusionQueryPrecise;
+    //VkBool32    pipelineStatisticsQuery;
+    //VkBool32    vertexPipelineStoresAndAtomics;
+    //VkBool32    fragmentStoresAndAtomics;
+    //VkBool32    shaderTessellationAndGeometryPointSize;
+    //VkBool32    shaderImageGatherExtended;
+    //VkBool32    shaderStorageImageExtendedFormats;
+    //VkBool32    shaderStorageImageMultisample;
+    //VkBool32    shaderStorageImageReadWithoutFormat;
+    //VkBool32    shaderStorageImageWriteWithoutFormat;
+    //VkBool32    shaderUniformBufferArrayDynamicIndexing;
+    //VkBool32    shaderSampledImageArrayDynamicIndexing;
+    //VkBool32    shaderStorageBufferArrayDynamicIndexing;
+    //VkBool32    shaderStorageImageArrayDynamicIndexing;
+    bool shaderClipDistance = false;
+    bool shaderCullDistance = false;
+    bool vertexAttributeDouble = false; // shaderFloat64
+    //VkBool32    shaderInt64;
+    //VkBool32    shaderInt16;
+    //VkBool32    shaderResourceResidency;
+    //VkBool32    shaderResourceMinLod;
+    //VkBool32    sparseBinding;
+    //VkBool32    sparseResidencyBuffer;
+    //VkBool32    sparseResidencyImage2D;
+    //VkBool32    sparseResidencyImage3D;
+    //VkBool32    sparseResidency2Samples;
+    //VkBool32    sparseResidency4Samples;
+    //VkBool32    sparseResidency8Samples;
+    //VkBool32    sparseResidency16Samples;
+    //VkBool32    sparseResidencyAliased;
+    //VkBool32    variableMultisampleRate;
+    bool inheritedQueries = false;
+
+    /* Vulkan 1.1 Core */
+    //VkBool32           storageBuffer16BitAccess;
+    //VkBool32           uniformAndStorageBuffer16BitAccess;
+    //VkBool32           storagePushConstant16;
+    //VkBool32           storageInputOutput16;
+    //VkBool32           multiview;
+    //VkBool32           multiviewGeometryShader;
+    //VkBool32           multiviewTessellationShader;
+    //VkBool32           variablePointersStorageBuffer;
+    //VkBool32           variablePointers;
+    //VkBool32           protectedMemory;
+    //VkBool32           samplerYcbcrConversion;
+    bool shaderDrawParameters = false;
+
+    /* Vulkan 1.2 Core */
+    //VkBool32           samplerMirrorClampToEdge;
+    bool drawIndirectCount = false;
+    //VkBool32           storageBuffer8BitAccess;
+    //VkBool32           uniformAndStorageBuffer8BitAccess;
+    //VkBool32           storagePushConstant8;
```

Comment:
for now make GL and GLES report `false` and leave a comment with a `TODO: implemenent, then return true`